ClothesApp/Application/Repositories/IBrandsRepository.cs

```diff
@@ -0,0 +1,14 @@
+using Domain.Entities;
+
+namespace Application.Repositories;
+
+public interface IBrandsRepository : IBaseRepository<Brand>
+{
+    Task<bool> DoesBrandExist(string name);
+
+    Task<bool> DoesBrandExist(long id);
+
+    Task<Brand> DeleteBrandById(long id);
+
+    Task<Brand> GetBrandById(long id);
```

Comment:
GetById looks like very common action to me that can be used and implemented for every entity